src/components/NoteDetails.js

```diff
@@ -0,0 +1,193 @@
+const Notes = {
+  '16': {
+    noteName: "C",
+    noteNameOctave: "C2",
+    isAccidental: false,
+    frequency: 65.40639,
+    string: "C"
+  },
+  // '17': {
+  //   noteName: "C-Sharp",
+  //   noteNameOctave: "C#2",
+  //   isAccidental: true,
+  //   frequency: 69.29566,
+  //   string: "C"
+  // },
+  '18': {
+    noteName: "D",
+    noteNameOctave: "D2",
+    isAccidental: false,
+    frequency: 73.41619,
+    string: "C"
+  },
+  '19': {
+    noteName: "E-Flat",
+    noteNameOctave: "D#2",
+    isAccidental: true,
+    frequency: 77.78175,
+    string: "C"
+  },
+  '20': {
+    noteName: "E",
+    noteNameOctave: "E2",
+    isAccidental: false,
+    frequency: 82.40689,
+    string: "C"
+  },
+  '21': {
+    noteName: "F",
+    noteNameOctave: "F2",
+    isAccidental: false,
+    frequency: 87.30706,
+    string: "C"
+  },
+  // '22': {
+  //   noteName: "F-Sharp",
+  //   noteNameOctave: "F#2",
+  //   isAccidental: true,
+  //   frequency: 92.49861,
+  //   string: "C"
+  // },
+  '23': {
+    noteName: "G",
+    noteNameOctave: "C2",
+    isAccidental: false,
+    frequency: 97.99886,
+    string: "G"
+  },
+  // '24': {
+  //   noteName: "A-Flat", // can also be G-Sharp
+  //   noteNameOctave: "G#2",
+  //   isAccidental: true,
+  //   frequency: 103.8262,
+  //   string: "G"
+  // },
+  '25': {
+    noteName: "A",
+    noteNameOctave: "A2",
+    isAccidental: false,
+    frequency: 110,
+    string: "G"
+  },
+  '26': {
+    noteName: "B-Flat", // can also be A-Sharp
+    noteNameOctave: "A#2",
+    isAccidental: true,
+    frequency: 116.5409,
+    string: "G"
+  },
+  '27': {
+    noteName: "B",
+    noteNameOctave: "B2",
+    isAccidental: false,
+    frequency: 123.4708,
+    string: "G"
+  },
+  '28': {
+    noteName: "C",
+    noteNameOctave: "C3",
+    isAccidental: false,
+    frequency: 130.8128,
+    string: "G"
+  },
+  // '29': {
+  //   noteName: "C-Sharp",
+  //   noteNameOctave: "C#3",
+  //   isAccidental: true,
+  //   frequency: 138.5913,
+  //   string: "G"
+  // },
+  '30': {
+    noteName: "D",
+    noteNameOctave: "D3",
+    isAccidental: false,
+    frequency: 146.8324,
+    string: "D"
+  },
+  // '31': {
+  //   noteName: "E-Flat", // can also be D-Sharp
+  //   noteNameOctave: "D#3",
+  //   isAccidental: true,
+  //   frequency: 155.5635,
+  //   string: "D"
+  // },
+  '32': {
+    noteName: "E",
+    noteNameOctave: "E3",
+    isAccidental: false,
+    frequency: 164.8138,
+    string: "D"
```

Comment:
should this be "E"?

Reply:
Nope. :-) The cello has four strings. They are called C, G, D, and A because that's what note they play when no fingers are pressing on them. How you play different notes is by pressing down at different spots on the strings. For this note, you would press down with your pointer finger on the D string and it would produce a E.